chia/util/file_keyring.py

```diff
@@ -244,6 +253,44 @@ def delete_password(self, service: str, user: str) -> None:
                     keys.pop(service)
                 self.write_keyring()
 
+    def get_label(self, fingerprint: int) -> Optional[str]:
+        """
+        Returns the label for the given fingerprint or None if there is no label assigned.
+        """
+        with self.lock_and_reload_if_required():
+            return self.cached_labels().get(fingerprint)
+
+    def set_label(self, fingerprint: int, label: str) -> None:
+        """
+        Set a label for the given fingerprint. This will force a write to keyring.yaml on success.
+        """
+        # First validate the label
+        if len(label.strip()) == 0:
```

Comment:
I would strip the label in case the user accidentally added a space or two before/after the name.

```suggestion
        clean_label = label.strip()
        if len(clean_label) == 0:
```

Reply:
The other checks (against the stripped label) look ok to me.

Reply:
See e906fcd265c2f463daa878724b217e581d9728d9